### PR TITLE
Don't run CI scripts which install software on CI machines

### PR DESCRIPTION
**Pull request by natemcmaster**
Next attempt to resolve https://github.com/aspnet/AspNetCore-Internal/issues/2210. Installing VS on the fly is extremely error prone. Let's work with the dnceng team to get updates to the VS build image instead of trying to do it on the fly.

### CODE REVIEW

**Comment by natemcmaster** [comment]
cref https://github.com/dotnet/core-eng/issues/5760 - the coreeng team is already working on updating the pools to VS 2019 RTM. In the meantime, I let's try to use the version of VS already available on agents.

**Comment by jkotalik** [comment]
Are the correct workloads available? Like C++ stuff?

**Review by JunTaoLuo — COMMENTED** [review]
Sure we can see if this works better.

**Comment by natemcmaster** [comment]
If they are not, we'll need to switch pools.

**Comment by JunTaoLuo** [comment]
Hmm 
```
F:\workspace\_work\1\s\build\tasks\RepoTasks.csproj : error MSB4236: The SDK 'Microsoft.NET.Sdk' specified could not be found.
```

**Comment by natemcmaster** [comment]
Yeah, sending bang mail to dnceng for help.

**Comment by natemcmaster** [comment]
Internal validation build: https://dev.azure.com/dnceng/internal/_build/results?buildId=150975

**Comment by dougbu** [comment]
@natemcmaster we really can't go back to the old -temp pools w/ AspNetCore. We were consistently seeing half-day delays for jobs that we started on weekday mornings. That's not as bad as everything failing but it's not something we want to endure again.

**Comment by natemcmaster** [comment]
@dougbu I'll take slower queue times over not meeting our 3.0 preview 4 release deadlines. Going back to temp pools is the best option we have available until we can get the BYOC pools up to parity with the temp images.

I would like to merge asap. The internal validation build of this change passed, code signing and all https://dev.azure.com/dnceng/internal/_build/results?buildId=150975, and we really need to get 3.0 p4 builds rolling.

**Comment by natemcmaster** [comment]
Looks like the dnceng team can reset BYOC machines fairly easy, and that they should be sufficient for our build. The failing builds from last night were left-over machines that were in a bad state. Let's try staying on BYOC for now.

**Comment by natemcmaster** [comment]
Spoke too soon: 

```
Detected Visual Studio Build Tools 2019 (16.0.28705.295) in 'C:\Program Files (x86)\Microsoft Visual Studio\2019\Preview'
>>> msbuild.exe F:\workspace\_work\1\s\build\tasks\RepoTasks.csproj -v:m -restore -t:Publish -p:Configuration=Release -p:PublishDir=F:\workspace\_work\1\s\build\tasks\bin\publish -nologo -p:RepoTasksSdkPath=F:\workspace\_work\1\s\.dotnet\buildtools\korebuild\3.0.0-build-20190322.1\scripts\..\msbuild\KoreBuild.RepoTasks.Sdk\Sdk /p:Configuration=Release /p:SignType= /p:BuildAllProjects=true /p:BuildNodeJS=false /p:BuildJava=false /p:_RunBuild=True /p:_RunPack=True /p:_RunTests=False /p:_RunSign=True /p:TargetArchitecture=arm /p:TargetOsName=win @F:\workspace\_work\1\s\artifacts\logs\msbuild.logger.rsp
F:\workspace\_work\1\s\build\tasks\RepoTasks.csproj : error MSB4236: The SDK 'Microsoft.NET.Sdk' specified could not be found.

```

**Comment by natemcmaster** [comment]
Retargeting to master. Will merge this after https://github.com/dotnet/core-eng/issues/5760 is complete, expected ETA - next week.

File: .azure/pipelines/jobs/default-build.yml
Comment (dougbu) [inline .azure/pipelines/jobs/default-build.yml]: Where is this parameter used? I suggest disabling overrides entirely unless absolutely necessary in one of the files in the parent directory.

File: .azure/pipelines/jobs/default-build.yml
Comment (dougbu) [inline .azure/pipelines/jobs/default-build.yml]: Are these pools new? What exactly do they / will they have on the agents?

File: .azure/pipelines/jobs/default-build.yml
Comment (dougbu) [inline .azure/pipelines/jobs/default-build.yml]: To confirm: Chrome will be installed by default? Cool&hellip;

**Review by dougbu — APPROVED** [review]

File: .azure/pipelines/jobs/default-build.yml
Comment (natemcmaster) [reply .azure/pipelines/jobs/default-build.yml]: Unused. Will remove.

File: .azure/pipelines/jobs/default-build.yml
Comment (natemcmaster) [reply .azure/pipelines/jobs/default-build.yml]: These pools already exist, but have VS 2019 Enterprise (and SQL Server).

File: .azure/pipelines/jobs/default-build.yml
Comment (natemcmaster) [reply .azure/pipelines/jobs/default-build.yml]: Should be after https://dnceng.visualstudio.com/internal/_git/dotnet-helix-machines/pullrequest/971?_a=overview is deployed

**Comment by natemcmaster** [comment]
Blocked on https://github.com/dotnet/core-eng/issues/6029

**Comment by natemcmaster** [comment]
This PR can't move forward without https://dev.azure.com/dnceng/internal/_git/dotnet-helix-machines/pullrequest/971, which was rejected. Moving to internal discussion to resolve inter-team differences.

**Pull request closed without merging**
